src/scripts/utils/counter.js

```diff
@@ -0,0 +1,15 @@
+import { tv } from '../API/TV-maze.js';
+
+class Counter {
+  countElements = (array) => array.length
+
+  moviesCounter = async () => {
+    const arrayOfShows = await tv.getAllShows();
+    return this.countElements(arrayOfShows);
+  }
+}
+
+const counter = new Counter();
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Please remove this eslint disable line its best to fix the issue in the code.